WordPress/openverse-frontend
# Update sketchfab viewer aspect ratio

## Fixes


Fixes #888 by @zackkrida 

## Description


Constrains the sketchfab viewer to the 1000px/500px (2/1) aspect ratio indicated by @panchovm in the comments in the linked issue.

## Testing Instructions

Check out this branch and run the app using `pnpm dev`. Then visit `localhost:8443/image/00a6a7a4-6605-4cba-bacf-a08b0da546b9` and verify that the viewer is the correct aspect ratio at various screen widths.

I thought about adding a snapshot test for this but wasn't sure about it. Would we mock the request to the sketchfab-viewer script and the subsequent sketchfab API requests for the model information? Seems thorny!

## Checklist

- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
